lib/ae_mdw/db/sync/contract.ex

```diff
@@ -15,7 +14,7 @@ defmodule AeMdw.Db.Sync.Contract do
 
   @spec create(pubkey(), pubkey(), tuple(), integer(), {integer(), integer()}) ::
           term() | :invalid_contract
-  def create(contract_pk, owner_pk, tx, txi, bi) do
+  def create(contract_pk, owner_pk, tx, txi, block) do
```

Comment:
it could receive the block_hash, that is only field used from the record.

Reply:
Good call. Done